app/views/application/_background.html.erb

```diff
@@ -0,0 +1,37 @@
+<%-
+  $background = "navbar-overlap"
+  $new_background = "navbar-overlap background"
+  $no_background = "navbar-overlap"
```

Comment:
Whats the purpose of having background already having the content of no_background and assigning it again? As it seems to have no background per default, it´s not necessary to set the "no-background" again. Maybe it would be a good idea to split this method into submethods (mobile desktop) to increase the readability.